### PR TITLE
Add swoole_error_log for swoole

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: reference/swoole/functions/swoole-error-log.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+
+<refentry xml:id="function.swoole-version" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (cmb69) [inline reference/swoole/functions/swoole-error-log.xml:4]:
```suggestion
<refentry xml:id="function.swoole-error-log" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment (sy-records) [reply reference/swoole/functions/swoole-error-log.xml]: done

Comment (cmb69) [reply reference/swoole/functions/swoole-error-log.xml]: Thanks!

File: reference/swoole/functions/swoole-error-log.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  &reftitle.description;
+  <methodsynopsis>
+   <type>void</type><methodname>swoole_error_log</methodname>
+   <methodparam><type>int</type><parameter>level</parameter></methodparam>
+   <methodparam><type>string</type><parameter>msg</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+
+  </para>
+
```
Comment (cmb69) [inline reference/swoole/functions/swoole-error-log.xml:20]: If it is not possible to document the function for now, please add:
```suggestion
  &warn.undocumented.func;
```

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR! However, it would be good to actually document the function, i.e. to describe the general purpose, the parameters (especially the `$level`; are there constant defined in swoole for that purpose?). Also, there should be return value section (see https://github.com/php/doc-en/pull/507/files#r603979680). If that is not possible, add the `&warn.undocumented.func;` entity reference, like shown below.

Furthermore, there should be an entry for the function in https://github.com/php/doc-en/blob/master/reference/swoole/versions.xml, which states the availability of this function (i.e. in which version of swoole it has been added).

**Comment by sy-records** [comment]
@cmb69 I modified it, can you review it again?

**Comment by cmb69** [comment]
Thanks you, especially for also documenting the constants! I'm going to add availability info to these, and will apply the patch.

**Comment by cmb69** [comment]
Applied as f03dfae1dcaf64d695189673e6db82aa3b7659a1.

**Pull request closed without merging**